test/controllers/HealthCheckControllerSpec.scala

```diff
@@ -1,38 +1,31 @@
 package controllers
 
 import org.mockito.Mockito._
-import org.scalatestplus.mockito.MockitoSugar
-import org.scalatestplus.play._
-import org.scalatestplus.play.guice._
-import play.api.mvc.ControllerComponents
-import play.api.mvc.Results.InternalServerError
 import play.api.test._
 import play.api.test.Helpers._
 import repositories.HealthCheckRepository
+import _stub.AbstractControllerSpec
 
-object HealthCheckControllerSpec extends PlaySpec with GuiceOneAppPerTest with MockitoSugar {
-  private val ccMock = mock[ControllerComponents]
+class HealthCheckControllerSpec extends AbstractControllerSpec {
   private val repoMock = mock[HealthCheckRepository]
   private val controller = new HealthCheckController(ccMock, repoMock)
 
-  private val failRepoMock = mock[HealthCheckRepository]
-  when(failRepoMock.canConnect).thenReturn(false)
-  private val failController = new HealthCheckController(ccMock, failRepoMock)
-
-  "HealthCheckController" must {
-    "web be valid" in {
+  describe("HealthCheckController") {
+    it("can get OK from web()") {
       val result = controller.web().apply(FakeRequest())
-      status(result) mustBe OK
+      status(result) shouldBe OK
```

Comment:
`mustBe` と `shouldBe`がただの雰囲気の違いってほんと。。？

> Trait MustMatchers is an alternative to Matchers that provides the exact same meaning, syntax, and behavior as Matchers, but uses the verb must instead of should. The two traits differ only in the English semantics of the verb: should is informal, making the code feel like conversation between the writer and the reader; must is more formal, making the code feel more like a written specification.

http://doc.scalatest.org/3.0.1-2.12/org/scalatest/MustMatchers.html

Reply:
そうです。文脈が自然になる方を使うだけ。
jestのitとtestが実質同じなのと一緒。